interactors/checkbox.js

```diff
@@ -0,0 +1,29 @@
+import { CheckBox, createInteractor, perform } from '@bigtest/interactor';
+
+export default createInteractor('checkbox')({
+  selector: 'div[class^=checkbox-]',
+  locator: (el) => el.querySelector('[class^=labelText]')?.textContent,
+  filters: {
+    ...CheckBox().specification.filters,
+    id: (el) => el.querySelector('input')?.id,
+    checked: (el) => el.querySelector('input')?.checked,
+    value: (el) => el.querySelector('input')?.value,
+    label: (el) => el.textContent,
+    ariaLabel: (el) => el.querySelector('input')?.ariaLabel,
+    ariaInvalid: (el) => el.querySelector('input')?.getAttribute('aria-invalid') === 'true',
+    feedbackText: (el) => el.querySelector('[role=alert]').textContent,
+    hasWarning: (el) => !!el.className.match(/hasWarning/),
+    hasError: (el) => !!el.className.match(/hasError/),
+    disabled: (el) => el.disabled // we have cases where it is undefined and then what?
+  },
+  actions: {
+    ...CheckBox().specification.actions,
```

Comment:
Are we sure this is what we want given that a checkbox wraps a single `input` element?

Reply:
Good point. Removed.